### PR TITLE
Added Parsec support to mbed-fcce

### PR DESCRIPTION
**Pull request by ygoyal18**
1. Renamed mbed-fcc to mbed-fcce
1. Compile fcce with parsec-se-driver when MBED_EDGE_CORE_CONFIG_PARSEC_TPM_SE_SUPPORT=ON

This is currently failing with build errors.

### CODE REVIEW

File: recipes-wigwag/mbed-fcce/mbed-fcce/mbed-fcce/0001-Added-trusted-storage-to-Yocto-target.patch [hunk 1]
```diff
@@ -0,0 +1,105 @@
+From 3904a9ccb468999c17883857af1869329713b155 Mon Sep 17 00:00:00 2001
+From: Yash Goyal <ygoyal@wigwag.com>
```
Comment (JanneKiiskila) [inline recipes-wigwag/mbed-fcce/mbed-fcce/mbed-fcce/0001-Added-trusted-storage-to-Yocto-target.patch:2]: wigwag email - should be using the @pelion.com email instead @ygoyal18 .
'

**Review by petedyerarm — APPROVED** [review]
LGTM

File: recipes-wigwag/mbed-fcce/mbed-fcce/mbed-fcce/0001-Added-trusted-storage-to-Yocto-target.patch (hunk 1 above)
Comment (ygoyal18) [reply recipes-wigwag/mbed-fcce/mbed-fcce/mbed-fcce/0001-Added-trusted-storage-to-Yocto-target.patch]: Fixed.

**Comment by ygoyal18** [comment]
Merging it to add-parsec-support so that we can directly clone the branch and figure out the build failure faster.

**Pull request merged**